air/src/execution_step/boxed_value/stream.rs

```diff
@@ -208,9 +210,13 @@ impl Stream {
     }
 
     /// Removes empty generations from current values.
-    fn remove_empty_generations(&mut self) {
+    pub(crate) fn remove_empty_generations(&mut self) {
```

Comment:
Thanks to the latest change in this PR, this function doesn't need to be public.

Reply:
Done